client/blocks/reader-subscription-list-item/connected.jsx

```diff
@@ -96,6 +96,7 @@ class ConnectedSubscriptionListItem extends Component {
 export default compose(
 	connect( ( state, ownProps ) => ( {
 		isFollowing: isFollowingSelector( state, { feedId: ownProps.feedId, blogId: ownProps.siteId } ),
+		url: ownProps.url.match( /^https?:\/\// ) ? ownProps.url : `http://${ ownProps.url }`,
```

Comment:
Should we preprend `https` vs. `http`?

Reply:
The site URL is checked by [parse_url](https://www.php.net/manual/en/function.parse-url.php). The actual scheme doesn't matter as far as subscribing to the feed is concerned, it just needs to be included.

Reply:
Got it, so this is just to pass the endpoint validation and then the url will be parsed before it's actually used.